src/Microsoft.AspNet.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -139,13 +139,13 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
                         var ticket = new AuthenticationTicket(principal, new AuthenticationProperties(), Options.AuthenticationScheme);
                         var validatedTokenContext = new ValidatedTokenContext(Context, Options)
                         {
-                            AuthenticationTicket = ticket
+                            Ticket = ticket
```

Comment:
Looks like the OIDC notifications will also need a parameter/property standardisation (not for this PR, of course)